odoo/addons/base/models/res_lang.py

```diff
@@ -88,8 +88,24 @@ def _register_hook(self):
         if not self.search_count([]):
             _logger.error("No language is active.")
 
-    @api.model
+    # TODO v14, remove me
     def load_lang(self, lang, lang_name=None):
+        _logger.warning("Call to deprecated method load_lang, use _create_lang or _activate_lang instead")
+        language = self._activate_lang(lang)
+        if not language:
+            self._create_lang(lang, lang_name=lang_name)
+
+    def _activate_lang(self, code=False):
+        """ Activate languages
+        :param code: optional code of the language to activate, using self if not specified
```

Comment:
That's a super weird API, I'd rather the method either always ignored its subject (and only used the code) *or* always used the code and there was a version of _lang_get (or even a modification on _lang_get itself) which allowed getting deactivated langs.